validation-test/StdlibUnittest/CrashingTests.swift

```diff
@@ -17,10 +27,13 @@ var TestSuiteCrashes = TestSuite("TestSuiteCrashes")
 
 TestSuiteCrashes.test("crashesUnexpectedly1") {
   print("crashesUnexpectedly1")
+  #if os(Windows)
+    Sleep(1)
+  #endif
   fatalError("This should crash")
```

Comment:
Can you wrap this pattern up in a helper function? `fatalErrorWithDelayIfNeeded` or something.

Reply:
I will.